src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableEntityContext.cs

```diff
@@ -141,11 +228,34 @@ void IDurableEntityContext.SetState(object o)
             this.StateWasAccessed = true;
         }
 
-        internal void Writeback()
+        internal void Writeback(out ResponseMessage stateSerializationFailedNotification)
```

Comment:
nit: consider renaming to `TryWriteback(...)` and having it return `bool`. That follows best practices for .NET API design. A return value of `false` also makes it more clear when to expect the `stateSerializationFailedNotification` to be non-null.

Reply:
Makes sense.